src/applications/vaos/containers/DateTimeRequestPage.jsx

```diff
@@ -1,89 +1,55 @@
 import React from 'react';
 import { connect } from 'react-redux';
-import SchemaForm from 'platform/forms-system/src/js/components/SchemaForm';
+import moment from 'moment';
+
 import {
-  openFormPage,
-  updateFormData,
+  onCalendarChange,
   routeToNextAppointmentPage,
   routeToPreviousAppointmentPage,
 } from '../actions/newAppointment.js';
 import { focusElement } from 'platform/utilities/ui';
 import { scrollAndFocus } from '../utils/scrollAndFocus';
 import FormButtons from '../components/FormButtons';
+import CalendarWidget from '../components/calendar/CalendarWidget';
 import { getFormPageInfo } from '../utils/selectors';
-import DateTimeRequestField from '../components/DateTimeRequestField';
+import { CALENDAR_INDICATOR_TYPES } from '../utils/constants';
 
 const pageKey = 'requestDateTime';
 const pageTitle = 'Choose a day and time for your appointment';
-
-const initialSchema = {
-  type: 'object',
-  required: ['calendarData'],
-  properties: {
-    calendarData: {
-      type: 'object',
-      properties: {
-        currentlySelectedDate: {
-          type: 'string',
-        },
-        currentRowIndex: {
-          type: 'number',
-        },
-        selectedDates: {
-          type: 'array',
-          items: {
-            type: 'object',
-            properties: {
-              date: {
-                type: 'string',
-              },
-              optionTime: {
-                type: 'string',
-                enum: ['AM', 'PM'],
-              },
-            },
-          },
-        },
-      },
+const missingDateError =
+  'Please select at least one preferred date for your appointment. You can select up to three dates.';
+
+export function getOptionsByDate() {
+  return [
+    {
+      value: 'AM',
+      label: 'AM',
     },
-  },
-};
-
-const uiSchema = {
-  calendarData: {
-    'ui:field': DateTimeRequestField,
-    'ui:title': 'What date and time would you like to make an appointment?',
-    'ui:options': {
-      hideLabelText: true,
+    {
+      value: 'PM',
+      label: 'PM',
     },
-  },
-};
-
+  ];
+}
 export class DateTimeRequestPage extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { validationError: null };
+    this.state = { submitted: false, validationError: null };
```

Comment:
I didn't notice this in my PR, but after adding `submitted`, we probably don't need `validationError` in local state, and could always call that method instead. But I don't think there's any harm in it for now and I don't want to make this PR bigger.